app/customer.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict
+import dataclasses
+import json
+import os
+
+
+@dataclasses.dataclass
+class Customers:
```

Comment:
Must be named into single
```suggestion
class Customer:
```